frontend/ui/widget/screenshoter.lua

```diff
@@ -86,11 +88,16 @@ function Screenshoter:onScreenshot(filename, when_done_func)
 end
 
 function Screenshoter:chooseFolder()
-    local screenshot_dir = G_reader_settings:readSetting("screenshot_dir") or DataStorage:getDataDir() .. "/screenshots/"
-    local confirm_box = ConfirmBox:new{
+    local screenshot_dir_default = ffiutil.realpath(DataStorage:getDataDir()) .. "/screenshots/"
+    local screenshot_dir = G_reader_settings:readSetting("screenshot_dir") or screenshot_dir_default
+    local confirm_box = MultiConfirmBox:new{
         text = T(_("Screenshot folder is set to:\n%1\n\nChoose a new folder for screenshots?"), screenshot_dir),
-        ok_text = _("Choose folder"),
-        ok_callback = function()
+        choice1_text = _("Default"),
```

Comment:
`Use default` ? as a verb, like the others?

Reply:
Yes, `Use default` is good.
Can it be `Choose new` symmetrically?

Reply:
(I'm no English doctor, but "Choose new" sounds gramatically dubious, choose new what ? :) and we can choose the same current folder.)